.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,7 +1,7 @@
 ---
 name: Bug report
-about: Create a report to help us improve
-title: ''
+about: Report a problem
+title: "[BUG]"
 labels: ''
 assignees: ''
 
```

Comment:
> Describe the bug
> A clear and concise description of what the bug is.

I've seen templates in other packages that ask:
```
Expected Behavior:
Actual Behavior:
```
Is there a place for that in the describe the bug section to help focus the information?